language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/FillRecordFieldsCodeAction.java

```diff
@@ -90,8 +89,12 @@ public List<CodeAction> getCodeActions(CodeActionContext context,
         if (expressionNode.isEmpty()) {
             return Collections.emptyList();
         }
-        CodeActionContextTypeResolver contextTypeResolver = new CodeActionContextTypeResolver(context);
-        Optional<TypeSymbol> typeSymbol = expressionNode.get().apply(contextTypeResolver);
+
+        Optional<TypeSymbol> typeSymbol = Optional.empty();
```

Comment:
you don't have to initialize the typeSymbol to Optional.empty. Alternatively, we can move the logic inside the if condition 94 to the initializer,  by doing an early return if `context.currentSemanticModel().isEmpty()  || context.currentDocument().isEmpty()`.